#!/usr/bin/env python
PACKAGE = "oshkosh"
import roslib;
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("WheelDrivePower",	double_t,	0,	"suspension spring",	2200,	0,  10000)	
gen.add("WheelDriveDamping",	double_t,	0,	"suspesion Damping",	100,	0,  1000)	

exit(gen.generate(PACKAGE, "oshkosh_wheels_drive_node", "oshkosh_wheels_drive"))
